app/views/documents/show/_attachments.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= render "govuk_publishing_components/components/summary_list", {
+  id: "attachments",
+  title: t("documents.show.attachments.title"),
+  borderless: true,
+  edit: (
+    if @edition.editable?
+      { href: attachments_path(@edition.document),
+        data_attributes: { gtm: "edit-tags" } }
+    end
+  ),
+  block: t("documents.show.attachments.no_attachments")
```

Comment:
We should probably remove this until we do the overall rendering work - it was a surprise to see it's still here when I've uploaded a featured attachment.